annotation/app/microservice_communication/search.py

```diff
@@ -193,6 +194,10 @@ def expand_response(
             "id": response_task.pop("job_id"),
             "name": job_names.get(task.job_id, None),
         }
+        response_task["user"] = {
+            "id": response_task.get("user_id"),
+            "name": user_logins.get(response_task.get("user_id")),
+        }
```

Comment:
Can we get an 'id' by response_task.user_id?
And 'name': userl_logins.response_task.user_id?

Reply:
No we can't - .user_id not always there.